documentation/references.md

```diff
@@ -15,13 +18,22 @@ If you applied AMICI in your work and your publication is missing, please let us
 <div id="ref-StaedterSch2021">
 <p>Städter, Philipp, Yannik Schälte, Leonard Schmiester, Jan Hasenauer, and Paul L. Stapor. 2021. “Benchmarking of Numerical Integration Methods for Ode Models of Biological Systems.” <em>Scientific Reports</em> 11 (1): 2696. <a href="https://doi.org/10.1038/s41598-021-82196-2">https://doi.org/10.1038/s41598-021-82196-2</a>.</p>
 </div>
+<div id="ref-StenPod2021">
+<p>Sten, Sebastian, Henrik Podéus, Nicolas Sundqvist, Fredrik Elinder, Maria Engström, and Gunnar Cedersund. 2021. “A Multi-Data Based Quantitative Model for the Neurovascular Coupling in the Brain.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2021.03.25.437053">https://doi.org/10.1101/2021.03.25.437053</a>.</p>
+</div>
+<div id="ref-VanhoeferMat2021">
+<p>Vanhoefer, Jakob, Marta R. a. Matos, Dilan Pathirana, Yannik Schälte, and Jan Hasenauer. 2021. “Yaml2sbml: Human-Readable and -Writable Specification of Ode Models and Their Conversion to Sbml.” <em>Journal of Open Source Software</em> 6 (61): 3215. <a href="https://doi.org/10.21105/joss.03215">https://doi.org/10.21105/joss.03215</a>.</p>
```

Comment:
```suggestion
<p>Vanhoefer, Jakob, Marta R. A. Matos, Dilan Pathirana, Yannik Schälte, and Jan Hasenauer. 2021. “yaml2sbml: Human-readable and -writable specification of ODE models and their conversion to SBML.” <em>Journal of Open Source Software</em> 6 (61): 3215. <a href="https://doi.org/10.21105/joss.03215">https://doi.org/10.21105/joss.03215</a>.</p>
```

Reply:
Is this generated automatically from the `.bib`? There, the title is capitalized as in the publication.

Reply:
Yes, it's auto-generated. With the current solution, it's unfortunately not easily possible to change the style. Maybe needs to be rewritten at some point.